lib/Slic3r/GUI/MainFrame.pm

```diff
@@ -631,6 +632,11 @@ sub select_view {
     $self->{plater}->select_view($direction);
 }
 
+sub zoom{
```

Comment:
This method is unneeded. Just call `$self->{plater}->zoom()` directly from the menu item handler.